cylc/flow/cfgspec/globalcfg.py

```diff
@@ -137,7 +137,7 @@
 
         .. versionadded:: 8.0.0
 
-           This is how ``timeout handler`` previously behaved.
+           Previously ``timeout`` functioned as a stall timeout.
```

Comment:
Not sure this is an improvement

Reply:
On changing the `timeout` item name to `workflow timeout` these docstrings no longer need to explain that `timeout` now means "workflow timeout" whereas it used to mean "stall timeout". Now plain `timeout` is obsolete, so I'm just noting, using this same sentence in every `stall timeout` item that stall timeout functionality used to be configured (Cylc 7) by the `timeout` item. Could still be worded better I'm sure, but we can tweak that later.  Yikes, convoluted.